exercises/set_intersection/python/main.py

```diff
@@ -0,0 +1,22 @@
+# Returns the intersection of two lists, tuples, or sets
+# Note True == 1 and False == 0
+def intersection(list1,list2):
+    try:
+        return set(list1) & set(list2)
```

Comment:
probably cleaner to explicitly use `... == 1` so that you return `True` or `False` instead of a number